src/channel.js

```diff
@@ -0,0 +1 @@
+module.exports = '__THEMING__';
```

Comment:
I would use something with a random key on it:
1. in order to prevent collisions
2. in order to make it even more clear to a user looking at the context object that this shouldn't be used directly.

```js
export default  `package-name-${Math.random().toString().substr(2, 6)}`
```

Btw. why not to use es6 export statement?